### PR TITLE
to shorten the build time by editng the pre built ISO

### LINKED ISSUE #74: Shorten the Build time for meilix generator by using mkiso
### Short description

by using mksio and squash tools instead of rebuilding whole iso which takes time we can create changes in iso and repack it

### PR DESCRIPTION
**Pull request by xeon-zolt**
### Short Description
I have:
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message Fixes #74
### For the reviewers
I have:
- [ ] Reviewed this pull request by an authorized contributor.
- [ ] The reviewer is assigned to the pull request.

### CODE REVIEW

**Comment by meets2tarun** [comment]
https://github.com/travis-ci/docs-travis-ci-com/issues/281

**Comment by meets2tarun** [comment]
Environment variable doesn't get pass in the fossasia/meilix CI test.

**Comment by xeon-zolt** [comment]
@meets2tarun it does not explains why I do not have same error on the Travis of my fork

**Comment by meets2tarun** [comment]
@xeon-zolt I'm looking for it and till yet just mentioned my finding.
One more thing to mention that this PR extracts fossasia/meilix
and your commit extracts xeon-zolt/meilix 
Are both on the meilix branch are even?

**Comment by xeon-zolt** [comment]
@meets2tarun  yes both branches are even only 2 files are edited as you can see in the pull request

**Comment by xeon-zolt** [comment]
@medicalwei thanks after -n flag on squashfs it do not shows error

**Comment by xeon-zolt** [comment]
@meets2tarun review it

**Comment by meets2tarun** [comment]
Shouldn't we are accepting a deployment of the iso?

File: build1.sh [hunk 1]
```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env bash
+#packages required to edit
+sudo apt-get install -qq squashfs-tools genisoimage
+#downloading the ISO to edit
+wget -q https://github.com/fossasia/meilix/releases/download/untagged-c9f68d2fa6b3d1cdbd99/meilix-zesty-20170611-i386.iso
```
Comment (meets2tarun) [inline build1.sh]: this untagged needs to be get changed to a stable and latest version @xeon-zolt

File: build1.sh [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+sudo cp /etc/hosts edit/etc/
+sudo mount --bind /dev/ edit/dev
+
+sudo chroot edit <<EOF
+
+# execute environment variable
+ls # to test the files if any new file is added
+chmod +x meilix-generator.sh
+echo "$(<meilix-generator.sh)" #to test the file
+./meilix-generator.sh
```
Comment (meets2tarun) [inline build1.sh]: Where can I find this script? @xeon-zolt

File: build1.sh (hunk 1 above)
Comment (xeon-zolt) [inline build1.sh]: will change it to lxqt once ISO once it is ready

File: build1.sh (hunk 2 above)
Comment (xeon-zolt) [inline build1.sh]: the script is to be generated by https://github.com/fossasia/meilix-generator/pull/76

**Pull request merged**